Package.swift

```diff
@@ -0,0 +1,24 @@
+// swift-tools-version:5.3
+import PackageDescription
+
+let packageVersion = "0.32.0"
+
+let package = Package(
+  name: "PayPalCheckout",
+  platforms: [
+    .iOS(.v11)
+  ],
+  products: [
+    .library(
+      name: "PayPalCheckout",
+      targets: ["PayPalCheckout"])
```

Comment:
Should `)` go to new line? Not sure about formatting rules for this file type. Looks like Swift, so we can probably follow our usual formatting.